calour/transforming.py

```diff
@@ -302,6 +300,12 @@ def center_log_ratio(exp: Experiment, method=lambda matrix: matrix + 1, centrali
     skbio.stats.composition.clr
     skbio.stats.composition.centralize
     """
+    try:
+        from skbio.stats.composition import clr, centralize as skbio_centralize
+    except ModuleNotFoundError as err:
+        logger.error('scikit-bio must be installed in order to use this function')
+        raise err
```

Comment:
why you need to try and capture exception?? you are raising the err anyway!

Reply:
In order to make better instructions on why it's failing - scikit-bio is not installed...
don't think it complicates too much but not critical - if you feel strongly about it i can delete...

Reply:
`ModuleNotFoundError: No module named 'dafasdf'` looks very clear to me. no need for extra code